src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -186,6 +186,7 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
         options['batch_size'] = batch_size
         options['max_iterations'] = max_iterations
         options['verbose'] = verbose
+        options['show_loss'] = False
```

Comment:
Could we use a leading underscore here, to make it more clear that this is (for now) an internal option?